### PR TITLE
Remove the usage of hardcoded mincpuplatform values

### PR DESCRIPTION
**Pull request by dimitdb**
I would like to no longer pass mincpuplatform in the call to create a VM. This is causing issues when CPI is asked to create gen2 VM (n2) in any of the following zones: us-central1-a and europe-west1-b. Here is the error, returned by the google API:

`googleapi: Error 400: The selected machine type (n2-custom-2-4096) has a required CPU platform of cascadelake. The minimum CPU platform must match this, but was broadwell.`

The need for having the map minCpuPlatform, that specifies min CPU platform "Intel Broadwell" for zones us-central1-a and europe-west1-b appears to be no longer valid. Based on the test, I've performed, the mincpuplatform in both zones now defaults to Intel Broadwell (not Sandy Bridge), when trying to create a gen1 VM.

### CODE REVIEW

**Comment by dbdimitrov83** [comment]
Related to https://github.com/cloudfoundry/bosh-google-cpi-release/issues/322

**Comment by dimitdb** [comment]
Related to [#322](https://github.com/cloudfoundry/bosh-google-cpi-release/issues/322)

File: src/bosh-google-cpi/google/instance_service/google_instance_service_create.go [hunk 1]
```diff
@@ -84,7 +84,8 @@ func (i GoogleInstanceService) Create(vmProps *Properties, networks Networks, re
 		Labels:            vmProps.Labels,
 		GuestAccelerators: acceleratorParams,
 		// Specify a non-Sandy Bridge CPU for known zones defined in the minCpuPlatform map
-		MinCpuPlatform: minCpuPlatform[vmProps.Zone],
```
Comment (friegger) [inline src/bosh-google-cpi/google/instance_service/google_instance_service_create.go:87]: This is the only place where minCpuPlatform is used. If there is no longer a scenario where a default CPU platform causes issues, I suggest removing minCpuPlatform entirely in this file and adjust the tests accordingly.

**Review by friegger — CHANGES_REQUESTED** [review]

File: src/bosh-google-cpi/google/instance_service/google_instance_service_create.go (hunk 1 above)
Comment (dimitdb) [reply src/bosh-google-cpi/google/instance_service/google_instance_service_create.go:87]: Removed the unnecessary code and comments. 
As far as I can see, only these two tests are relevant to the changes (both are passing, meaning we do not get Sandy Bridge CPU)
https://github.com/cloudfoundry/bosh-google-cpi-release/blob/076c3f8ee5e0250a8b7a8cb32e637ef6dd3f20b9/src/bosh-google-cpi/integration/vm_test.go#L288
https://github.com/cloudfoundry/bosh-google-cpi-release/blob/076c3f8ee5e0250a8b7a8cb32e637ef6dd3f20b9/src/bosh-google-cpi/integration/vm_test.go#L327

Both are worth keeping the way they are I think.

Comment (friegger) [reply src/bosh-google-cpi/google/instance_service/google_instance_service_create.go:87]: Let's at least at a comment to the test why those tests exist or extend the description, indicating that in the past Sandy was the minimum provided platform and the agent is incompatible.

Comment (dimitdb) [reply src/bosh-google-cpi/google/instance_service/google_instance_service_create.go:87]: Added comment and extended the tests description.

**Review by ramonskie — APPROVED** [review]
lgtm

**Review by friegger — APPROVED** [review]

**Comment by friegger** [comment]
@ramonskie should this PR go against the develop branch?

**Comment by rkoster** [comment]
Thanks! @dimitdb

**Pull request merged**